src/komodo_structs.cpp

```diff
@@ -135,15 +135,15 @@ std::ostream& operator<<(std::ostream& os, const event& in)
  * @param pos the starting position (will advance)
  * @param data_len full length of data
  */
-event_pubkeys::event_pubkeys(uint8_t* data, long& pos, long data_len, int32_t height) : event(EVENT_PUBKEYS, height)
+event_pubkeys::event_pubkeys(uint8_t* data, long& pos, long data_len, int32_t height) : event_pubkeys(height)
```

Comment:
This change is a part of https://github.com/KomodoPlatform/komodo/pull/510, right? May be we should separate line [#138](https://github.com/KomodoPlatform/komodo/pull/481/files#diff-10bcb1d8edc16456d7b98fedb3834fc3adf81d945d59963596beba41781530b4R138) and line [#146](https://github.com/KomodoPlatform/komodo/pull/481/files#diff-10bcb1d8edc16456d7b98fedb3834fc3adf81d945d59963596beba41781530b4R146) from this PR? Of course, it's important changes, but i would like to consider them in other PR as it's not related to NPOINTS refactoring.